blockchain/validate.go

```diff
@@ -733,7 +755,7 @@ func (b *BlockChain) checkBlockContext(block *bchutil.Block, prevNode *blockNode
 	// A block must not exceed the maximum allowed block payload when
 	// serialized.
 	serializedSize := block.MsgBlock().SerializeSize()
-	if serializedSize > maxBlockSize {
+	if uint64(serializedSize) > maxBlockSize {
```

Comment:
Same with this one, seems those shouldn't need changing.

Reply:
Same as above.